docs/playwright.md

```diff
@@ -87,7 +89,7 @@ It will wait for a small amount of time (100ms) by default after each user actio
 
 ## Writing Tests
 
-CodeceptJS test should be created with `gt` command:
+Additional CodeceptJS tests should be created with `gt` command:
```

Comment:
There is 1 test created by the generator, which seems to be the suggested setup flow.